tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -829,7 +829,7 @@
   },
   {
     "id": "{{.ok}} deleted action {{.name}}\n",
-    "translation": "{{.ok}} deleted action {{.name}}\n"
+    "translation": "{{.ok}} deleted action '{{.name}}'\n"
```

Comment:
Also looks like the keys (id) will be out of sync with the values (translation).